convert2rhel/checks.py

```diff
@@ -67,6 +73,84 @@ def perform_pre_ponr_checks():
+        "gpgcheck=1\n"
+        "enabled=1\n"
+        "sslcacert=%s\n"
+        "gpgkey=file://%s\n" % (ssl_cert_path, gpg_path)
+    )
+
+    cmd = [
+        "repoquery",
+        "--releasever=%s" % system_info.version.major,
+    ]
+
+    repo_dir = tempfile.mkdtemp(prefix="convert2rhel_repo.", dir=utils.TMP_DIR)
+    utils.mkdir_p(repo_dir)
+    repo_path = os.path.join(repo_dir, "convert2rhel.repo")
+    store_content_to_file(filename=repo_path, content=repo_content)
+
+    cmd.append("--setopt=reposdir=%s" % repo_dir)
+    cmd.append("convert2rhel")
+
+    raw_output_convert2rhel_versions, return_code = run_subprocess(cmd, print_output=False)
+
+    if return_code != 0:
+        logger.warning(
+            "Couldn't check if this is the latest version of convert2rhel\n"
+            "repoquery failed %s, %s" % (return_code, raw_output_convert2rhel_versions)
+        )
+        return
+
+    PKG_NEVR = r"\b(\S+)-(?:([0-9]+):)?(\S+)-(\S+)\b"
+    convert2rhel_versions = re.findall(PKG_NEVR, raw_output_convert2rhel_versions, re.MULTILINE)
+    latest_version = ("0", "0.00", "0")
+
+    for package_version in convert2rhel_versions:
+        # rpm.lableCompare(pkg1, pkg2) compare two kernel version strings and return
+        # -1 if str2 is greater then str1, 0 if they are equal, 1 if str1 is greater the str2
+        ver_compare = rpm.labelCompare(package_version[1:], latest_version)
+        if ver_compare > 0:
+            latest_version = package_version[1:]
+
+    ver_compare = rpm.labelCompare(("0", convert2rhel_version, "0"), ("0", latest_version[1], "0"))
+    if ver_compare < 0:
+        if "CONVERT2RHEL_UNSUPPORTED_VERSION" in os.environ:
+            logger.warning(
+                "You are currently running %s and the latest version of convert2rhel is %s.\n"
+                "'CONVERT2RHEL_UNSUPPORTED_VERSION' environment detected, continuing conversion"
+                % (convert2rhel_version, latest_version[1])
+            )
+
+        else:
+            if int(system_info.version.major) <= 6:
+                logger.warning(
+                    "You are currently running %s and the latest version of convert2rhel is %s.\n"
+                    "Only the latest version is supported for conversion." % (convert2rhel_version, latest_version[1])
+                )
+
+            else:
+                logger.critical(
+                    "You are currently running %s and the latest version of convert2rhel is %s.\n"
+                    "Only the latest version is supported for conversion. If you want to ignore"
+                    " this you can set 'CONVERT2RHEL_UNSUPPORTED_VERSION' to continue"
+                    % (convert2rhel_version, latest_version[1])
+                )
+
+    else:
+        logger.info(
+            "You are currently running %s and the latest version of convert2rhel is %s.\n"
+            "Continuing conversion." % (convert2rhel_version, latest_version[1])
+        )
```

Comment:
Since this will be either the same version or we have a newer one installed, I would instead opt for a debug log message saying "Latest available convert2rhel version is installed"

Reply:
ok, sounds good I'll go ahead and get that changed!

Reply:
@SpyTec also me and Toshio weren't sure where to put the cert files, at the moment we have put them in the data dir. let me know if there is a better place to put it?

Reply:
I believe that the current place is fine. It is in the right directory, but the question is more about should it go to a subdirectory. Given it's a single file I would say no and keep it where it is

Reply:
ok, sounds good! Thank you for going over that for me!

Reply:
I have made changes to this making it a debug message, and changing the message to be clear as well!